services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DynamoDbEnhancedClient.java

```diff
@@ -88,11 +88,29 @@ static Builder builder() {
     /**
      * The builder definition for a {@link DynamoDbEnhancedClient}.
      */
-    interface Builder {
+    interface Builder extends DynamoDbEnhancedResource.Builder {
+        /**
+         * The regular low-level SDK client to use with the enhanced client.
+         * @param dynamoDbClient an initialized {@link DynamoDbClient}
+         */
         Builder dynamoDbClient(DynamoDbClient dynamoDbClient);
 
-        Builder extendWith(DynamoDbEnhancedClientExtension dynamoDbEnhancedClientExtension);
-
+        /**
+         * {@inheritDoc}
```

Comment:
nit: `{@inheritDoc}` is not necessay. `@Override` automatically renders the inherited doc.

Reply:
oh! I'll whip that out then. Thanks.